apps/wallet/frontend/src/components/CreateAllocation.tsx

```diff
@@ -130,51 +127,44 @@ const CreateAllocation: React.FC = () => {
                 })
               }
             />
+            <Typography variant="h6">Requested at</Typography>
+            <TextField
```

Comment:
Why do we want a text field instead of a date picker? Because you need it to be more precise? Which timezone do you enter it in?

Reply:
probably just add a comment on this